GetARock/GetARock/Screen/AddGathering/AddGatheringViewController.swift

```diff
@@ -42,7 +46,38 @@ class AddGatheringViewController: UIViewController {
     // MARK: - Method
 
     @IBAction func cancelButtonAction(_ sender: UIBarButtonItem) {
-        dismiss(animated: true)
+        if introductionTextView.text.count > 0 || (titleTextField?.text ?? "").count > 0 || gatheringLocation != nil {
+            let cancelAlert = UIAlertController(title: nil, message: "작성중인 내용이 있습니다. 작성을 취소하시겠습니까?", preferredStyle: .alert)
+            let cancel = UIAlertAction(title: "아니오", style: .cancel)
+            let confirm = UIAlertAction(title: "예", style: .destructive, handler: {_ in
+                self.dismiss(animated: true)
+            })
+            cancelAlert.addAction(cancel)
+            cancelAlert.addAction(confirm)
+            self.present(cancelAlert, animated: true)
+        } else {
+            dismiss(animated: true)
+        }
+    }
+
+    @IBAction func saveButtonAction(_ sender: UIBarButtonItem) {
+        if let errorString = invalidInputErrorString() {
+            let alert = UIAlertController(title: nil, message: errorString, preferredStyle: .alert)
+            let confirm = UIAlertAction(title: "확인", style: .default)
+            alert.addAction(confirm)
+            self.present(alert, animated: true)
+        } else {
+            let gatheringAddTestGathering = Gathering(title: titleTextField.text ?? "이름없음",
+                                                        host: MockData.bands[0], // 테스트용, 추후 변경
+                                                        status: .recruiting,
+                                                        date: dateTimePicker.date,
+                                                        location: gatheringLocation ?? Location(name: "Default", address: "defaultAddress", additionalAddress: "defaultAdditionalAddress", coordinate: Coordinate(latitude: 36.01900, longitude: 129.34370)),
+                                                        introduction: introductionTextView.text,
+                                                        createdAt: Date())
```

Comment:
p3;

마찬가지로 줄바꿈이 필요해보여요! 

```suggestion
            let gatheringAddTestGathering = Gathering(
                title: titleTextField.text ?? "이름없음",
                host: MockData.bands[0], // 테스트용, 추후 변경
                status: .recruiting,
                date: dateTimePicker.date,
                location: gatheringLocation ?? Location(
                    name: "Default",
                    address: "defaultAddress",
                    additionalAddress: "defaultAdditionalAddress",
                    coordinate: Coordinate(latitude: 36.01900, longitude: 129.34370)
                ),
                introduction: introductionTextView.text,
                createdAt: Date()
            )
```

Reply:
위의 줄바꿈과 같은 커밋에서 반영했습니다. 9382c3fa45041a2e35c1ede461d10559275629f4